src/tests/topp/ProteomicsLFQ_4_out.mzTab

```diff
@@ -88,7 +88,7 @@ PRT	O76015|KRT38_HUMAN	Keratin, type I cuticular Ha8 - Homo sapiens (Human).	nul
 PRT	O76014|KRT37_HUMAN	Keratin, type I cuticular Ha7 - Homo sapiens (Human).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.038461538461538	null	null	0.015590200445434	null	null	null	null	null	null	null	null	null	null	null	null	null	1	0	protein_details
 PRT	O76013|KRT36_HUMAN	Keratin, type I cuticular Ha6 - Homo sapiens (Human).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.038461538461538	null	null	0.014989293361884	null	null	null	null	null	null	null	null	null	null	null	null	null	1	0	protein_details
 PRT	P00761|TRYP_PIG	Trypsin - Sus scrofa (Pig).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.052631578947368	P00761|TRYP_PIG	null	0.077922077922078	2.0	0.0	0.0	2.0	null	null	0.0	null	null	0.0	null	null	null	null	null	single_protein
-PRT	P02769|ALBU_BOVIN	Serum albumin - Bos taurus (Bovine).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.052631578947368	P02769|ALBU_BOVIN	590-UNIMOD:4	0.401976935749588	91.0	0.0	0.0	91.0	null	null	0.0	null	null	0.0	null	null	null	null	null	single_protein
+PRT	P02769|ALBU_BOVIN	Serum albumin - Bos taurus (Bovine).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.052631578947368	P02769|ALBU_BOVIN	590-UNIMOD:4	0.401976935749588	71.0	0.0	0.0	71.0	null	null	0.0	null	null	0.0	null	null	null	null	null	single_protein
```

Comment:
this is wrong ... quantities need to be summed over fractions if we report it like this